src/components/structures/LoggedInView.js

```diff
@@ -585,7 +585,8 @@ const LoggedInView = createReactClass({
                 limitType={usageLimitEvent.getContent().limit_type}
             />;
         } else if (this.props.showCookieBar &&
-            this.props.config.piwik
+            this.props.config.piwik &&
+            navigator.doNotTrack != 1
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack claims it'll be "1" so maybe `!== "1"`

Reply:
(I know `!= 1` works too but we prefer `!==`)